### DISCUSSION TITLE
More attribution for Floorp?

### DISCUSSION THREAD

**Discussion by TheDanikReal** [goastian/midori-desktop#70] [General]
Midori project relies on Floorp, many features are taken from Floorp, and I don't see any attribution for Floorp developers in Midori (only on this github repository, but i don't think that many people go here). What about adding something like "Midori project is a fork of Floorp browser with significant changes" in the "about Midori" page where version is displayed?

**Comment by Ponchale** [comment]
We give the recognition publicly, in addition to in the readme, we also respect the license, nothing forces us to place a reference in the or, in fact we do not place based on Firefox that we also place in the readme.

I will close the discussion because it is something meaningless and without purpose.

**Comment by Ponchale** [comment]
Ubuntu doesn't say in every part of your system or on its website, "based on Debian" or Manjaro, Manjaro doesn't say in every part of your system or in its about. "Based on Arch."

Neither does Nextcloud https://nextcloud.com/about/ friends, stop resentment and hatred, the differences between Floorp and Midori are at the level of orientation Floorp is based on advanced functions, Midori on being private, fast and secure as well as light ( light in the consumption of resources, not in the weight of the program) In addition, an entire ecosystem of applications is being built around Midori, all free software and open source.
